stable/nginx-ingress/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.6.18
+version: 1.6.19
```

Comment:
Lets bump the minor version for this change to 1.7.0

Reply:
Changed version to 1..0